tools/check-git-version.js

```diff
@@ -0,0 +1,25 @@
+const shell = require('shelljs');
+
+function checkGitVersion() {
+  try {
+    const gitVersion = shell
+      .exec('git --version', { silent: true })
```

Comment:
Should we use `simple-git` for this because what simple-git uses is what is important?

Reply:
We will get the same o/p string while using shell/simple-git, I do not see any extra benefit it using Simple Git, sorry if  I am missing something..

Reply:
Please use simple-git then

Reply:
Sure